build.gradle.kts

```diff
@@ -163,28 +119,26 @@ tasks.withType<JavaCompile>().configureEach {
 // Build Config
 
 buildConfig {
+    useJavaOutput()
+    packageName("com.mongodb.hibernate.internal")
     buildConfigField("NAME", provider { project.name })
     buildConfigField("VERSION", provider { "${project.version}" })
-    packageName("com.mongodb.hibernate")
 }
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 // Dependencies
 
 dependencies {
-    listOf(libs.junit.jupiter, libs.assertj, libs.logback.classic).forEach {
-        testImplementation(it)
-        integrationTestImplementation(it)
-    }
-
+    testImplementation(libs.bundles.test.common)
     testImplementation(libs.mockito.junit.jupiter)
-    testRuntimeOnly(libs.junit.platform.launcher)
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
```

Comment:
I'd be interested in the reasons to remove this dependency from libs.versions.toml.

Reply:
Since v9, Gradle would require explicit spec of the platform otherwise a forward compatibility warning will be issued (you can check your branch to see).

So in the current user guide, the following pattern always shows up (e.g. https://docs.gradle.org/current/userguide/java_testing.html#compiling_and_executing_junit_jupiter_tests):
```
dependencies {
    testImplementation("org.junit.jupiter:junit-jupiter:5.7.1")
    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
}
```
Given it is not using version, it seems inappropriate to config the launcher in toml file. That is why I excluded it.